cmd/eudico/mirvalidator/run.go

```diff
@@ -81,7 +81,14 @@ var runCmd = &cli.Command{
 		&cli.IntFlag{
 			Name:  "segment-length",
 			Usage: "The length of an ISS segment. Must not be negative",
-			Value: 1,
+		},
+		&cli.DurationFlag{
+			Name:  "blocks-delay",
+			Usage: "The maximum delay in seconds between two blocks",
+		},
+		&cli.IntFlag{
```

Comment:
Something similar happens with this, I am wary of setting it as a flag instead of a compilation parameter or something "more set in stone"

Reply:
My changes are to decouple CMD with consensus logic and not store the parameters in the command configs.

Reply:
What do you mean by decoupling CMD with consensus logic? And what are command configs?
If there is an easy and clean way to not expose (at least for now) the `ConfigOffset` parameter, that might be a better choice. But it's a weak opinion based on weak understanding of how the parametrization works, so it's completely up to you.